risczero/src/benches/big_sha2.rs

```diff
@@ -1,6 +1,7 @@
 use rand::{rngs::StdRng, RngCore, SeedableRng};
-use risc0_zkp::core::sha::Digest;
-use risc0_zkvm::{Prover, Receipt};
+use risc0_zkvm::prove::Prover;
+use risc0_zkvm::receipt::Receipt;
```

Comment:
I'd prefer we use the top-level re-exported symbols `risc0_zkvm::{Prover, Receipt]`